src/resources/purchases.js

```diff
@@ -44,12 +46,34 @@ class Purchases extends ResourceBase {
     })
   }
 
-  async buyerConfirmReceipt(address) {
-    return await this.contractFn(address, "buyerConfirmReceipt")
+  async buyerConfirmReceipt(address, data={}) {
+    const review = await this._buildReview(data)
+    const args = [review.rating, review.ipfsHashBytes]
+    return await this.contractFn(address, "buyerConfirmReceipt", args)
+  }
+
+  async sellerGetPayout(address, data={}) {
+    const review = await this._buildReview(data)
+    const args = [review.rating, review.ipfsHashBytes]
+    return await this.contractFn(address, "sellerCollectPayout", args, {gas: 100000})
   }
 
-  async sellerGetPayout(address) {
-    return await this.contractFn(address, "sellerCollectPayout")
+  async _buildReview(data={}){
```

Comment:
@DanielVF Maybe we should consider using a json schema here instead of doing our own validation?

We're already doing this for user profile: https://github.com/OriginProtocol/origin-js/blob/master/src/resources/users.js#L19